cpu/imxrt117x-cm4/example/blinky.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+cd ../lib
+make clean all
+cd ../example
+make clean all
+
+pushd ../../../../_build/armv7m4-imxrt117x/prog.stripped/
+
+xxd -ps -c1 blinky.bin | awk '{printf("0x%s,",$0)}' > blinky.h
```

Comment:
```suggestion
xxd -ps -c1 blinky.bin | awk '{printf("0x%s,",$0);if (NR % 16 == 0) printf("\n")}' > blinky.h
```
More readable `blinky.h`, 16 bytes per line.

Reply:
blinky.h doesn't really need to be readalbe, but I changed it to the new proposed version